tldw_Server_API/tests/_plugins/media_fixtures.py

```
@@ -0,0 +1,52 @@
+    cleanup_test_file,
+)
+
+
+def _yield_file(path: str) -> Generator[str, None, None]:
+    try:
+        yield path
+    finally:
+        try:
+            cleanup_test_file(path)
+        except Exception:
+            pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)